department-of-veterans-affairs/caseflow
# adding flash to edit page

Connects #9729 

![image](https://user-images.githubusercontent.com/577629/53847704-89a4be00-3f66-11e9-8268-a16ae185d741.png)

Validation:
* create intake, when adding issues verify that banner does not show up
* edit intake, verify that when modifying issues the banner shows up
